plugin.go

```diff
@@ -58,32 +58,14 @@ var (
 // For a of predefined ranges, see the the [readme]
 // [readme]: https://github.com/JasonLovesDoggo/caddy-defender#embedded-ip-ranges
 type Defender struct {
-	// Ranges specifies IP ranges to block, which can be either:
-	// - CIDR notations (e.g., "192.168.1.0/24")
-	// - Predefined service keys (e.g., "openai", "aws")
-	// Default:
-	Ranges []string `json:"ranges,omitempty"`
-
-	// An optional whitelist of IP addresses to exclude from blocking. If empty, no IPs are whitelisted.
-	// NOTE: this only supports IP addresses, not ranges.
-	// Default: []
-	Whitelist []string `json:"whitelist,omitempty"`
-
-	// Message specifies the custom response message for 'custom' responder type.
-	// Required only when using 'custom' responder.
-	Message string `json:"message,omitempty"`
-
-	// RawResponder defines the response strategy for blocked requests.
-	// Required. Must be one of: "block", "garbage", "custom"
-	RawResponder string `json:"raw_responder,omitempty"`
-
-	// ServeIgnore specifies whether to serve a robots.txt file with a "Disallow: /" directive
-	// Default: false
-	ServeIgnore bool `json:"serve_ignore,omitempty"`
-	// responder is the internal implementation of the response strategy
-	responder responders.Responder
-	ipChecker *ip.IPChecker
```

Comment:
Please add the comments back. Caddy's documentation system fetches these to display on the documentation site.

The size of this struct doesn't really matter too much as there should only be one instance of it of a time as it's always passed by reference.

Reply:
besides that, LGTM!

Reply:
I had it locally, but didn't push 🙈 

Sorry about that. Good catch!